src/app/components/big-input/big-input.component.scss

```
@@ -0,0 +1,28 @@
+.container {
+  position: relative;
+
+  .input {
+    border: 2px solid #3bb5cc;
+    border-radius: 120px;
+    font-size: 20px;
+    outline: none;
+    padding: 20px 28px 20px 65px;
+    width: 100%;
+
+    &::-webkit-search-cancel-button{
+      background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' fill='%23777'><path d='M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z'/></svg>");
+      cursor: pointer;
+      height: 24px;
+      margin-left: .4em;
+      width: 24px;
+      -webkit-appearance: none;
+    }
+  }
+
+  &::before {
+    content: url("../../../assets/images/svg/magnifying-glass.svg");
```

Try use path absolute 
```suggestion
    content: url("/assets/images/svg/magnifying-glass.svg");
```